Header 	header
float32[]  radius
float32[]  angle
float32[] value
